src/components/ui4/ButtonUi4.tsx

```diff
@@ -196,9 +210,9 @@ const getStyles = cacheStyles((theme: Theme) => {
       // Reduce the bounds of a tertiary button so it doesn't appear to be too
```

Comment:
tertiary buttons should not have any difference in spacing. On some styles they may actually be filled with borders. Please create a task to fix in 4.1

Reply:
This was a request by Madison